clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -114,6 +117,69 @@ def start_clipper(
+            """
+            if (config_path):
+                with open(config_path, "r") as stream:
+                    config = yaml.load(stream)
+                    if ("links" in config):
+                        for link in config["links"]:
+                            try:
+                                self.link_model_to_app(
+                                    config["links"][link]["app_name"],
+                                    config["links"][link]["model_name"])
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                    if ("applications" in config):
+                        for application in config["applications"]:
+                            try:
+                                input_type = config["applications"][
+                                    application]["input_type"]
+                                slo = config["applications"][application][
+                                    "slo"]
+                                default_value = config["applications"][
+                                    application]["default_value"]
+                                self.register_application(
+                                    application, input_type, default_value,
+                                    slo)
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                    if ("models" in config):
+                        for model in config["models"]:
+                            try:
+                                config_image = config["models"][model]["image"]
+                                input_type = config["models"][model][
+                                    "input_type"]
+                                version = config["models"][model]["version"]
+                                labels = None
+                                if ("labels" in config["models"][model]):
+                                    labels = config["models"][model]["labels"]
+                                num_replicas = 1
+                                if ("num_replicas" in config["models"][model]):
+                                    num_replicas = config["models"][model][
+                                        "num_replicas"]
+                                batch_size = -1
+                                if ("batch_size" in config["models"][model]):
+                                    batch_size = config["models"][model][
+                                        "batch_size"]
+                                self.deploy_model(model, version, input_type,
+                                                  config_image, labels,
+                                                  num_replicas, batch_size)
+                            except ClipperException as e:
```

Comment:
Catch `KeyError`s and throw an appropriate `ClipperException`

Reply:
Done